src/node.cc

```diff
@@ -840,7 +819,12 @@ bool DomainsStackHasErrorHandler(const Environment* env) {
       return false;
 
     Local<Object> domain = domain_v.As<Object>();
-    if (DomainHasErrorHandler(env, domain))
+
+    Local<Value> has_error_handler = domain->Get(
+        env->context(), OneByteString(env->isolate(),
+                                      "_hasErrorListener")).ToLocalChecked();
+
+    if (has_error_handler->BooleanValue())
```

Comment:
`->IsTrue()`, this overload is deprecated (or use the overload that takes a context argument, but tha shouldn’t be necessary since we know the passed type is a boolean)